sql/size_utils.sql

```diff
@@ -13,22 +13,15 @@
 -- index_bytes        - Disc space used by indexes
 -- toast_bytes        - Disc space of toast tables
 -- total_bytes        - Total disk space used by the specified table, including all indexes and TOAST data
--- table_size         - Pretty output of table_bytes
--- index_bytes        - Pretty output of index_bytes
--- toast_bytes        - Pretty output of toast_bytes
--- total_size         - Pretty output of total_bytes
 
 CREATE OR REPLACE FUNCTION hypertable_relation_size(
     main_table              REGCLASS
 )
 RETURNS TABLE (table_bytes BIGINT,
                index_bytes BIGINT,
                toast_bytes BIGINT,
-               total_bytes BIGINT,
-               table_size  TEXT,
-               index_size  TEXT,
-               toast_size  TEXT,
-               total_size  TEXT) LANGUAGE PLPGSQL VOLATILE
+               total_bytes BIGINT
+               ) LANGUAGE PLPGSQL VOLATILE
                SECURITY DEFINER SET search_path = ''
```

Comment:
Does not need SECURITY DEFINER (nor SET search_path = '' because that's only set because it's security definer)